File: C#/Strategy/PaymentStrategys/PaypalPaymentStrategy.cs
using Strategy.Interfaces;

namespace Strategy.PaymentStrategys;

public sealed class PaypalPaymentStrategy : IPaymentStrategy
{
    /// <summary>
    /// Paypal payment processing
    /// </summary>
    /// <param name="amount"><inheritdoc/></param>
    public void ProcessPayment(decimal amount)
        => Console.WriteLine($"Processing paypal payment for {amount}");
}

File: PHP/Patterns/src/Behavioral/Strategy/PaymentWays/PaypalPaymentStrategy.php
<?php

namespace DesignPatterns\Behavioral\Strategy\PaymentWays;

use DesignPatterns\Behavioral\Strategy\Interfaces\PaymentStrategy;

class PaypalPaymentStrategy implements PaymentStrategy
{
    public function processPayment(float $amount): void
    {
        print("Processing payment of paypal...");
        print("Calculating fees of the amount" . $amount . "for paypal...");

    }
}
